package.json

```diff
@@ -7,5 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.18.2"
```

Comment:
It's a good practice to lock down the versions of your dependencies to ensure the stability of your application. Consider removing the caret (^) before the version number. [medium]